xlive/Blam/Engine/effects/particle_property.h

```diff
@@ -1,6 +1,8 @@
 #pragma once
 #include "Blam/Engine/math/function_definitions.h"
 
+struct s_particle_state;
```

Comment:
use a void pointer instead of forward declaring the type here?

Reply:
forward declaration to prevent reference issues and compilation issues, void* causes it to throw errors unless things are casted when called.